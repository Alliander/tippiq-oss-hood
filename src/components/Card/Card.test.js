import React from 'react';
import renderer from 'react-test-renderer';
import Element from './index';

jest.mock('../../piwik', () => ({
  push: jest.fn(),
}));

xtest('Renders', () => {
  const component = renderer.create(
    <Element />
  );
  const json = component.toJSON();
  expect(json).toMatchSnapshot();
});

