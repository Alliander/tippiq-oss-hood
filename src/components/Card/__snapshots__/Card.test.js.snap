exports[`test Renders 1`] = `
<div
  className={undefined}>
  <div
    className="undefined horizontal undefined  "
    onBlur={[Function]}
    onFocus={[Function]}
    tabIndex={0}>
    <div
      className={undefined}>
      <div
        aria-hidden={false}
        className={undefined}
        tabIndex={-1}>
        <div
          className={undefined}>
          <div
            className={undefined}
            style={
              Object {
                "backgroundImage": "url(undefined)",
              }
            }>
            <span
              className={undefined}>
               km
            </span>
          </div>
          <div
            className={undefined}>
            <div
              className={undefined}>
              <h4
                className={undefined}>
                <a
                  href={undefined} />
              </h4>
            </div>
          </div>
          <div
            className="undefined undefined">
            <span
              className={undefined} />
            <span
              className={undefined}>
              <button
                className={undefined}
                onClick={[Function]}>
                <i
                  className={undefined} />
              </button>
            </span>
          </div>
        </div>
      </div>
      <div
        aria-hidden={true}
        className={undefined}
        tabIndex={-1}>
        <div
          className={undefined}>
          <div
            className="undefined ">
            <span
              className={undefined} />
            <span
              className={undefined}>
              <button
                className={undefined}
                onClick={[Function]}>
                <i
                  className={undefined} />
              </button>
            </span>
          </div>
          <div
            className="undefined" />
        </div>
      </div>
    </div>
  </div>
</div>
`;
